core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -626,7 +628,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     producer.close()
   }
 
-  @Test
+  @Test(expected = classOf[org.apache.kafka.common.errors.TimeoutException])
```

Comment:
Same comment. If a test case did not previously timeout, we shouldn't change it.